build.gradle

```diff
@@ -28,14 +27,14 @@ repositories {
 group 'org.wiremock'
 
 allprojects {
-  version = "0.11.0"
+  version = "1.0.0-beta.1"
 
-  sourceCompatibility = 11
-  targetCompatibility = 11
+  sourceCompatibility = 17
+  targetCompatibility = 17
 
   ext {
     versions = [
-      wiremock: "3.13.2",
+      wiremock: "4.0.0-beta.17",
```

Comment:
Probably worth bumping this to .19 before merging as there have been some significant changes, including a module rename

Reply:
done